### ISSUE TITLE
Problem with network drives

### ISSUE THREAD

**Issue by LucaVazz** [johnpapa/lite-server#42]
Hey everyone,
I try to run a _lite-server_ instance which is supposed to serve files from a network-connected NTFS-drive.

[This](https://github.com/LucaVazz/lite-server_test) is my setup. It is based o the [README-Guide](https://github.com/johnpapa/lite-server/blob/master/README.md).

### The Problem
At first, it starts fine. But after about 8 to 10 seconds, lite-server crashes.

Running `npm install lite-server -g` didn't help.

If i try to run it from a local harddrive, it works completely fine.

###### Console:
```
U:\lite_network-test>npm run dev

> lite-server_network-test@0.0.1 dev U:\lite_network-test
> lite-server

Did not detect a `bs-config.json` or `bs-config.js` override file. Using lite-server defaults...
** browser-sync options **
{ injectChanges: false,
  files: [ './**/*.{html,htm,css,js}' ],
  server: { baseDir: './', middleware: [ [Function], [Function] ] } }
[BS] Access URLs:
 ---------------------------------------
       Local: http://localhost:3000
       External: http://172.27.209.218:3000
 ---------------------------------------
          UI: http://localhost:3001
          UI External: http://172.27.209.218:3001
 ---------------------------------------
[BS] Serving files from: ./
[BS] Watching files...
16.03.01 10:55:21 304 GET /index.html
events.js:154
      throw er; // Unhandled 'error' event
      ^

Error: watch null UNKNOWN
    at exports._errnoException (util.js:856:11)
    at FSEvent.FSWatcher._handle.onchange (fs.js:1296:21)

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "dev"
npm ERR! node v5.7.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! lite-server_network-test@0.0.1 dev: `lite-server`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the lite-server_network-test@0.0.1 dev script 'lite-server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lite-server_network-test package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     lite-server
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lite-server_network-test
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lite-server_network-test
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     U:\lite_network-test\npm-debug.log
```

###### npm-debug.log
```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'dev' ]
2 info using npm@3.6.0
3 info using node@v5.7.0
4 verbose run-script [ 'predev', 'dev', 'postdev' ]
5 info lifecycle lite-server_network-test@0.0.1~predev: lite-server_network-test@0.0.1
6 silly lifecycle lite-server_network-test@0.0.1~predev: no script for predev, continuing
7 info lifecycle lite-server_network-test@0.0.1~dev: lite-server_network-test@0.0.1
8 verbose lifecycle lite-server_network-test@0.0.1~dev: unsafe-perm in lifecycle true
9 verbose lifecycle lite-server_network-test@0.0.1~dev: PATH: C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin;U:\lite_network-test\node_modules\.bin;C:\Program Files\ConEmu;C:\Program Files\ConEmu\ConEmu;C:\ProgramData\Oracle\Java\javapath;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\System32\WindowsPowerShell\v1.0\;C:\windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Box\Box Edit\;C:\Program Files (x86)\IXOS\bin;C:\Program Files\nodejs\;C:\Users\luca.vazzano\Programs\apache-maven-3.3.3\bin;%USERPROFILE%\.dnx\bin;C:\Program Files\Microsoft DNX\Dnvm\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WebEx\Productivity Tools;C:\Program Files\setlX;C:\Program Files\TortoiseGit\bin;C:\Program Files (x86)\Sennheiser\SoftphoneSDK\;C:\Users\luca.vazzano\AppData\Roaming\npm;C:\Users\luca.vazzano\AppData\Local\Pandoc\
10 verbose lifecycle lite-server_network-test@0.0.1~dev: CWD: U:\lite_network-test
11 silly lifecycle lite-server_network-test@0.0.1~dev: Args: [ '/d /s /c', 'lite-server' ]
12 silly lifecycle lite-server_network-test@0.0.1~dev: Returned: code: 1  signal: null
13 info lifecycle lite-server_network-test@0.0.1~dev: Failed to exec dev script
14 verbose stack Error: lite-server_network-test@0.0.1 dev: `lite-server`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\lifecycle.js:232:16)
14 verbose stack     at emitTwo (events.js:100:13)
14 verbose stack     at EventEmitter.emit (events.js:185:7)
14 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\spawn.js:24:14)
14 verbose stack     at emitTwo (events.js:100:13)
14 verbose stack     at ChildProcess.emit (events.js:185:7)
14 verbose stack     at maybeClose (internal/child_process.js:827:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
15 verbose pkgid lite-server_network-test@0.0.1
16 verbose cwd U:\lite_network-test
17 error Windows_NT 6.1.7601
18 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "dev"
19 error node v5.7.0
20 error npm  v3.6.0
21 error code ELIFECYCLE
22 error lite-server_network-test@0.0.1 dev: `lite-server`
22 error Exit status 1
23 error Failed at the lite-server_network-test@0.0.1 dev script 'lite-server'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the lite-server_network-test package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     lite-server
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs lite-server_network-test
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls lite-server_network-test
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by LucaVazz** [comment]
I have lite server version _2.1.0_, nodejs version _5.7.0_ and npm version _3.6.0_ installed.

**Comment by cgmartin** [comment]
@LucaVazz the fix from #57 may help. Give `lite-server v2.2.0` a try.

**Comment by LucaVazz** [comment]
@cgmartin thank you very much! Now it works perfectly :smile_cat:
